src/scripts/imageList.ts

```diff
@@ -0,0 +1,169 @@
+export const imageList = [
```

Comment:
Curious though; how is this being utilized?

Reply:
That is a temporary image path used to generate the path stored in FE-assets. 
It needs to be deleted. Nice catch!